Source/WebKit/GPUProcess/graphics/WebGPU/RemoteCommandEncoder.cpp

```diff
@@ -38,17 +39,14 @@
 #include "WebGPUObjectHeap.h"
 #include <WebCore/WebGPUCommandEncoder.h>
 
-#if PLATFORM(COCOA)
 #define MESSAGE_CHECK(assertion) do { \
-    if (UNLIKELY(!(assertion))) { \
-        if (auto connection = m_gpuConnectionToWebProcess.get()) \
-            connection->terminateWebProcess(); \
-        return; \
+    if (auto didPassMessageCheck = !!(assertion)) { \
```

Comment:
Maybe the !! is confusing me, but doesn't this do nothing if the assertion is false?

Reply:
`!!` being effectively a conversion to bool operator, it would appear to be the case indeed which seems unintentional

Reply:
Oh oops, you're right. I meant to say !(assertion) here.